lib/ui/viewer/file/zoomable_image.dart

```diff
@@ -54,23 +54,15 @@ class _ZoomableImageState extends State<ZoomableImage>
   bool _loadedLargeThumbnail = false;
   bool _loadingFinalImage = false;
   bool _loadedFinalImage = false;
-  ValueChanged<PhotoViewScaleState>? _scaleStateChangedCallback;
-  bool _isZooming = false;
   PhotoViewController _photoViewController = PhotoViewController();
+  double? initialScale;
+  double percentage = 42.5 / 100;
```

Comment:
What's this [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming))? 👀

Reply:
It's just percentage value to detect, if the image is zoomed or not from its initial scale value, because for each image, we are calculating the scale by some calculations and then assigning the initial scale value. And if that scale value is less than  the current scale value, then the image is zoomed. So just added a percentage value to check the increase value.

But just now, got another solution. Will update that soon